app/models/user.rb

```diff
@@ -3,7 +3,7 @@ class User < ApplicationRecord
   has_many :comments, dependent: :destroy, foreign_key: :author_id
   has_many :likes, dependent: :destroy, foreign_key: :author_id
   validates :name, presence: true
-  #validates :posts_counter, numericality: { greater_than_or_equal_to: 0 }
+  # validates :posts_counter, numericality: { greater_than_or_equal_to: 0 }
```

Comment:
https://github.com/ismailakinkunmi/blog/blob/405f87f87909824be07abd5fa725b68118b269a2/app/controllers/posts_controller.rb#L1-L11

- [ ] Great job 👏🏻  however you have to make sure your app solves N+1 problem when fetching all posts and their comments for a user
- Example in the posts#index actions on line 4 you should use:
  `@posts = @author.posts.includes(:comments)` instead of `@posts = @author.posts `

- Note: Using includes we could also load associations from Comment if they existed. As an example, if Comment had a belongs_to :user association, we could load it like so:
 `@posts = Post.includes(comments: [:user])`

For more documentation, you can visit the [Solving the N+1 problem in Rails](https://medium.com/idopterlabs/solving-the-n-1-problem-in-rails-b5ea677e78e0) site

Reply:
@BenMukebo your feedback is not helpful here and doesn't give any room for me to improve on my code because when I tried ur hint is just like creating bug to work on, am available anytime for improvement, we can schedule 10 to 30 mins on zoom to look into it.

Reply:
@ismailakinkunmi, let's have a zoom call, I need to know why you got a bug!! just tell me when you're available!!